Sources/VariadicsGenerator/VariadicsGenerator.swift

```diff
@@ -129,14 +132,43 @@ struct VariadicsGenerator: ParsableCommand {
       emitTupleStruct(arity: arity)
     }
 
-    for arity in minArity...maxArity {
-      for permutation in Permutations(arity: arity) {
-        emitConcatenation(permutation: permutation)
+    print("Generating concatenation overloads...", to: &standardError)
+    for (leftArity, rightArity) in Permutations(totalArity: maxArity) {
+      print(
+        "  Left arity: \(leftArity)  Right arity: \(rightArity)",
+        to: &standardError)
+      emitConcatenation(leftArity: leftArity, rightArity: rightArity)
+    }
+
+    for arity in 0..<maxArity {
+      emitConcatenationWithEmpty(leftArity: arity)
+    }
+
+    output("\n\n")
+
+    print("Generating quantifiers...", to: &standardError)
+    for arity in 0..<maxArity {
+      print("  Arity \(arity): ", terminator: "", to: &standardError)
+      for kind in QuantifierKind.allCases {
+        print("\(kind.rawValue) ", terminator: "", to: &standardError)
+        emitQuantifier(kind: kind, arity: arity)
       }
-      output("\n\n")
+      print(to: &standardError)
     }
 
-    output("// END AUTO-GENERATED CONTENT")
+    output("\n\n")
+
+    output("// END AUTO-GENERATED CONTENT\n")
+
+    print("Done!", to: &standardError)
+  }
+
+  func tupleType(arity: Int, genericParameters: () -> String) -> String {
+    assert(arity >= 0)
+    if arity == 0 {
+      return genericParameters()
+    }
+    return "Tuple\(arity)<\(genericParameters())>"
```

Comment:
Is this the PR that drops the nominal public types, or a different one?

Reply:
Ah, are we dropping them for literal but not result builder?

Reply:
The public types, e.g. `Concatenation_0_1`, will be dropped in a follow-up PR.